tests/test_build_texinfo.py

```diff
@@ -53,6 +53,12 @@ def test_texinfo(app, status, warning):
     # now, try to run makeinfo over it
     cwd = os.getcwd()
     os.chdir(app.outdir)
+    with open('SphinxTests.texi') as texi_file:
+        # make sure anchors are sorted
+        texi_source = texi_file.read(8000)
```

Comment:
We usually use `path.text()` to read the content on testing scripts. Then you don't need to open a file.
Could you use `(app.outdir / 'SphinxTests.texi').text(encoding='utf8')` instead?